crash/src/app/view-accidents-container/view-accidents/view-accidents.component.scss

```diff
@@ -0,0 +1,31 @@
+
+
+.title{
```

Comment:
classes look good Mounika, but we're using tailwindcss for our stylings, so we'll try to avoid on creating classes in the scss files. so instead of doing `<div class="title"></div>` we can go

`<div class='text-black text-lg text-left [...]> </div>`

You can refer to Tailwind css stylings from their documentation: https://tailwindcss.com/docs/text-align

Basically TailwindCSS is like bootstrap but offers more flexibility and customization